src/components/UI/ExternalLink/ExternalLink.tsx

```diff
@@ -7,8 +7,12 @@ interface ExternalLinkProps {
 
 export const ExternalLink = ({ name, href }: ExternalLinkProps) => {
   return (
-    <a href={href} target="_blank" rel="noopener noreferrer">
-      {name}
-    </a>
+    <>
```

Comment:
why fragment?

Reply:
![Screenshot 2020-03-24 at 4 09 25 PM](https://user-images.githubusercontent.com/4774314/77402606-dfeed400-6de9-11ea-8c8d-2fa9ddb1d884.png)

if i nv put i getting this warning from ts

Reply:
ok i found out the problem, i should use `children: React.ReactNode;` instead of `children: ReactElement;`, will update